gandi/cli/commands/domain.py

```diff
@@ -50,12 +50,14 @@ def info(gandi, resource):
               help='Technical contact handle.')
 @click.option('--bill', default=None,
               help='Billing contact handle.')
+@click.option('--nameserver', default=None,
```

Comment:
`--nameservers` would be more consistant with the return shown as an example in the issue, and with the API param, wouldn't it?